src/Home/Home.tsx

```diff
@@ -35,15 +44,20 @@ function Home({ clusters }: HomeProps) {
         if (!perBuild[buildName]) {
           perBuild[buildName] = emptyValues();
         }
+        if (!perBuildMetadata[buildName]) {
+          perBuildMetadata[buildName] = emptyMetadata();
+        }
         total.standingBy += gsb.status.currentStandingBy ? gsb.status.currentStandingBy : 0;
         perCluster[clusterName].standingBy += gsb.status.currentStandingBy ? gsb.status.currentStandingBy : 0;
         perBuild[buildName].standingBy += gsb.status.currentStandingBy ? gsb.status.currentStandingBy : 0;
         total.active += gsb.status.currentActive ? gsb.status.currentActive : 0;
         perCluster[clusterName].active += gsb.status.currentActive ? gsb.status.currentActive : 0;
         perBuild[buildName].active += gsb.status.currentActive ? gsb.status.currentActive : 0;
+        perBuildMetadata[buildName].namespace = gsb.metadata.namespace ? gsb.metadata.namespace : "";
```

Comment:
Hello and thanks for looking into the project! I was out yesterday, so I couldn't explain some stuff.

So, here when I group the data by buildName it's actually from all clusters, that means that if two different clusters have a build with the same name the data gets aggregated, so linking to a specific cluster wouldn't be correct. The idea of this is that the user might have the same title in different regions and wants to see an overall status.

Looking back, the UI does not make it clear that we are aggregating the data like this, so there is some room for improvement here. A simple solution would be to just split the item into the actual builds from each cluster and change the key to something like `[buildName]:[cluster]`, but of course feel free to try something else.

Reply:
@javier-op - I have made the changes as per our discussion.